tests/resolvers/Mutation/forgotPassword.spec.ts

```diff
@@ -1,9 +1,15 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Do not use these comments for the whole file. Try to avoid using them, and if absolutely necessary, use them for particular lines.